snippets/pyaloha/event.py

```diff
@@ -46,3 +53,8 @@ def from_event(cls, event):
             event.event_time, event.user_info,
             event.data_list, event.data_list_len
         )
+
+    def __dumpdict__(self):
+        d = super(DictEvent, self).__dumpdict__()
+        d.update(self.data)
```

Comment:
It updates dict d with all fields (event params) of the DictEvent instance got from aloha KeyPairsEvent. Event class doesn't bother with event params mostly because it is used for aloha KeyEvents mostly